src/clr-angular/forms/select/select.spec.ts

```diff
@@ -99,6 +115,18 @@ export default function(): void {
                 expect(focused.classList.contains("clr-select-input")).toBe(true);
             });
 
+            it("focuses the input when the menu is clicked", () => {
+                ifOpenService.open = true;
+
+                const menu = context.clarityElement.querySelector(".menu");
+
+                menu.click();
+
+                const focused = context.clarityElement.querySelector(":focus");
+
+                expect(focused.classList.contains("clr-select-input")).toBe(true);
```

Comment:
Small mistake: https://github.com/vmware/clarity/blob/master/CODING_GUIDELINES.md#unit-testing
> Unit tests should not duplicate coverage. In particular, avoid multiple unit tests failing for the same error.

You're indirectly re-testing the `focusInput()` method here. You want to spy on this method, and expect it to have been called.

EDIT: I just noticed you have the right test a few lines below. Not sure what this one is for then.

Reply:
Yep. I added the Host click listener in the last commit and forgot to remove this test. Now the new test that you mentioned below tests for this. Will remove this test.